Tests/TuistSupportTests/System/SystemTests.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+import Basic
+import TuistSupport
+
+class SystemTests: XCTestCase {
```

Comment:
I was unsure about adding these tests since they are more integration tests and rely on an external state being set, I can remove them if we think they are not worth keeping.

Reply:
I think it's a good idea Would you mind moving them to [this integration test case class](https://github.com/tuist/tuist/blob/master/Tests/TuistSupportIntegrationTests/System/SystemIntegrationTests.swift)

Reply:
I've done that

Reply:
:+1: